### PR TITLE
fix(editor): Shift+Tab de-indents selection by one unit

### LINKED ISSUE #1430: Shift-Tab Behaviour
Whilst #1424 fixed the tab behaviour for indenting a selection, on the latest master (`v0.0.15-317-g935444c6 `) the inverse operation shift-tab doesn't work to dedent the selection

The behaviour in the current notebook is for tab to indent the selection 4 spaces and shift-tab to dedent 4 spaces. It would be great if the shift-tab operation were also implemented in nteract.

The GIF below shows the behaviour of nteract master vs the notebook:
tab2

### PR DESCRIPTION
**Pull request by peggyrayzis**
Fixes #1430 😎

### CODE REVIEW

**Comment by dhirschfeld** [comment]
I can confirm that on this PR both tab and shift-tab behave as expected (as discussed in #1430)!

File: src/notebook/components/cell/codemirror/index.js [hunk 1]
```diff
@@ -151,6 +151,7 @@ const CodeMirrorWrapper: CodeMirrorHOC = (EditorView, customOptions = null) =>
         extraKeys: {
           'Ctrl-Space': 'autocomplete',
           Tab: this.executeTab,
+          'Shift-Tab': editor => editor.execCommand('indentLess'),
```
Comment (rgbkrk) [inline src/notebook/components/cell/codemirror/index.js:154]: If you want this to be covered, you could probably do:

```js
'Shift-Tab': editor.execCommand.bind(editor, 'indentLess')
```

Comment (rgbkrk) [reply src/notebook/components/cell/codemirror/index.js:154]: That or use lodash's partial.

**Comment by jdetle** [comment]
I just merged it without attempting to fix the coverage issue that @rgbkrk mentioned, my bad @peggyrayzis!

**Comment by peggyrayzis** [comment]
No worries, I can make another PR!

**Pull request merged**